helpers/utils.py

```diff
@@ -461,3 +461,16 @@ def simplify_date(datetime_str: str) -> str:
     if datetime_str is None:
         return None
     return datetime_str.split("T")[0]
+
+
+def download_file(download_url: str, destination_path: str) -> None:
```

Comment:
Cool!

I have a feeling that I came across this function or one if its variants at least 2 
 times in the code base. But for the life of me, I couldn't tell ya where!